core/credential/credential.go

```diff
@@ -10,8 +10,8 @@ import (
 	"github.com/juju/names/v5"
 )
 
-// ID represents the id of a cloud credential.
-type ID struct {
+// Key represents the natural key of a cloud credential.
+type Key struct {
 	// Cloud is the cloud name that the credential applies to. Id is valid when
```

Comment:
"Id". There will be other places also I am pretty sure where code comments need to be updated.